CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - Stored payload metric per container (#2116)
 - Stored payload metric per shard (#2023)
 - Histogram metrics for RPC and engine operations (#2351) 
+- SN's version is announced via the attributes automatically but can be overwritten explicitly (#2455)
```

Comment:
Doesn't look correct here.

Reply:
took from the master branch for conflict prevention

Reply:
You can drop it.